src/Components/ABDM/LinkAbhaNumber/CreateWithAadhaar.tsx

```diff
@@ -0,0 +1,618 @@
+
+function VerifyAadhaar({ memory, setMemory, next, goTo }: IVerifyAadhaarProps) {
+  const { t } = useTranslation();
+  const [otp, setOtp] = useState("");
+
+  const validateMobileNumber = () => {
+    const phone = memory?.mobileNumber.replace("+91", "").replace(/ /g, "");
+    if (phone?.length !== 10) {
+      setMemory((prev) => ({
+        ...prev,
+        validationError: t("mobile_number_validation_error"),
+      }));
+      return false;
+    }
+
+    return true;
+  };
+
+  const handleSubmit = async () => {
+    if (!validateMobileNumber()) return;
+
+    setMemory((prev) => ({ ...prev, isLoading: true }));
+
+    const { res, data, error } = await request(
+      routes.abdm.healthId.abhaCreateVerifyAadhaarOtp,
+      {
+        body: {
+          otp: otp,
+          transaction_id: memory?.transactionId,
+          mobile: memory?.mobileNumber.replace("+91", "").replace(/ /g, ""),
+        },
+      },
+    );
+
+    if (res?.status === 200 && data) {
+      setMemory((prev) => ({
+        ...prev,
+        transactionId: data.transaction_id,
+        abhaNumber: data.abha_number,
+      }));
+      Notify.Success({
+        msg: data.detail ?? t("otp_verification_success"),
+      });
+      if (
+        data.abha_number.mobile ===
+        memory?.mobileNumber.replace("+91", "").replace(/ /g, "")
+      ) {
+        goTo(4); // skip linking mobile number
+      } else {
+        next();
+      }
+    } else {
+      Notify.Error({
+        msg: data?.detail ?? error?.message ?? t("otp_verification_error"),
+      });
+    }
+
+    setMemory((prev) => ({ ...prev, isLoading: false }));
+  };
+
+  return (
+    <div>
+      <div className="flex flex-col justify-center">
+        <TextFormField
+          name="aadhaar-number"
+          label={t("aadhaar_number")}
+          min={12}
+          max={16}
+          inputClassName="text-black tracking-[0.3em] font-bold placeholder:font-normal placeholder:tracking-normal text-center"
+          placeholder={t("enter_aadhaar_number")}
+          disabled={true}
+          value={memory?.aadhaarNumber}
+          onChange={({ value }) =>
+            setMemory((prev) => ({ ...prev, aadhaarNumber: value }))
+          }
+        />
+        <span
+          className={classNames(
+            "ml-2 text-sm font-medium text-gray-600",
+            !memory?.validationError && "-mt-4",
+          )}
+        >
+          {t("aadhaar_number_will_not_be_stored")}
+        </span>
+      </div>
+
+      <div className="mt-4">
+        <OtpFormField
+          name="otp"
+          onChange={(value) => setOtp(value as string)}
+          value={otp}
+          label={t("enter_aadhaar_otp")}
+          disabled={memory?.isLoading}
+        />
+      </div>
+
+      <div className="mt-0">
+        <label
+          htmlFor="mobile_number"
+          className="block text-base font-normal text-secondary-900"
+        >
+          {t("enter_mobile_number")}
+        </label>
```

Comment:
Pass this in the label prop of the below PhoneNumberFormField component, and use the `labelClassName` prop to update the classnames if necessary.